source/stylesheets/screen.css.scss

```diff
@@ -2,6 +2,8 @@
 @import 'normalize';
 @import 'variables';
 @import 'icon-font';
+@import 'font-awesome-sprockets';
+@import 'font-awesome';
```

Comment:
I can use a icomoon icon that is the equivalent of the font awesome one. Can you give me direction on how to go about regenerating the font files? I already grabbed what I needed to update the font-selection.json file. I could probably do the svg instead with the icomoon tools if that is generally preferred and it sounds simpler.